modules/tournament/src/main/TournamentScheduler.scala

```diff
@@ -496,7 +465,8 @@ Thank you all, you rock!""".some,
     ).flatten.filter(_.schedule.at.isAfter(rightNow))
 
   private def atTopOfHour(rightNow: LocalDateTime, hourDelta: Int): LocalDateTime =
-    rightNow.plusHours(hourDelta).withMinute(0)
+    val withHours = rightNow.plusHours(hourDelta)
+    LocalDateTime.of(withHours.date, LocalTime.of(withHours.getHour, 0))
```

Comment:
Calling out this important change.  `atTopOfHour` was not resetting seconds/nanos, causing tourneys to have extra scatter caused by when they got scheduled.  This "fixes" the function, but means that we now are fully reliant on the scatter explicitly added in this PR.